src/test/test_time_of_flight.cxx

```diff
@@ -47,10 +47,10 @@ START_NAMESPACE_STIR
 //!
 class cache_index{
 public:
-    cache_index() {
+    cache_index():
```

Comment:
this doesn't seem to change anything at all, but best to use constructor initialisers. I don't see where `cache_index` is used by the way

Reply:
I did this for codacy.

Reply:
great. but is this class actually used?(sorry, can't ee it)